model_compression_toolkit/ptq/pytorch/quantization_facade.py

```diff
@@ -112,16 +115,20 @@ def pytorch_post_training_quantization(in_module: Module,
                                                target_resource_utilization=target_resource_utilization,
                                                tb_w=tb_w)
 
-        tg = ptq_runner(tg, representative_data_gen, core_config, DEFAULT_PYTORCH_INFO, fw_impl, tb_w)
+        float_graph = copy.deepcopy(tg)
+
+        graph_with_stats_correction = ptq_runner(tg, representative_data_gen, core_config, DEFAULT_PYTORCH_INFO, fw_impl, tb_w)
```

Comment:
Please put DEFAULT_PYTORCH_INFO into a variable and use the variable same way as in the Keras facade?